SPECS/xapi-storage.spec

```diff
@@ -103,13 +103,14 @@ cp storage/api/__init__.py storage/api/datapath.py storage/api/volume.py storage
 
 %files ocaml-plugin-devel
 %defattr(-,root,root,-)
+%exclude %{_libdir}/ocaml/xapi-storage/*.ml
```

Comment:
Still, the addition of this line belongs in the other patch. Even more so now that the related line below is removed in that commit.